app/models/notification.rb

```diff
@@ -0,0 +1,55 @@
+class Notification < ActiveRecord::Base
+  belongs_to :activity, inverse_of: :notifications, dependent: :destroy
+
+  acts_as_readable on: :updated_at
+
+  def self.notify(user, course, object, options = {})
+    include_public_activity(object)
+    unless options.key?(:activity)
+      notify_only(user, course, object, options)
+      return
+    end
+    activity_and_notify(user, course, object, options)
+  end
+
+  class << self
+    private
+
+    def include_public_activity(object)
```

Comment:
Is this necessary? That will cause every active records have this gem's module...